lib/resources/package.rb

```diff
@@ -19,8 +19,10 @@ class Package < Inspec.resource(1)
         its('version') { should eq 1.9.5 }
       end
     "
-
-    def initialize(package_name, opts = {}) # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/CyclomaticComplexity
+    # rubocop:disable Metrics/PerceivedComplexity
+    def initialize(package_name, opts = {})
```

Comment:
In the future we usually just cascade the rubocop disables on the method call. That way we don't have to disable after. Example:
```
def initialize(package_name, opts = {}) # rubocop:disable Metrics/AbcSize Metrics/CyclomaticComplexity  Metrics/PerceivedComplexity
...
end
```